backend/src/main/java/com/example/demo/StatusEnum.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo;
+
+public enum StatusEnum {
+
+    OPEN("OPEN"),
```

Comment:
Die Strings hier brauchst du nicht. Du kannst unten auch `this` in das switch Statement werfen.